examples/dog-app/README.md

```diff
@@ -0,0 +1,6 @@
+# Dog app
+
+<p class="description">A basic Toolpad app that shows how to call an API</p>
+
+The app can be accessed from:
+Todo: Add codesandbox link
```

Comment:
Do we need to leave this reminder in the `README.md`? We can add this folder when the link is ready instead?

Reply:
I added the link; it should work when this PR is merged.